website/docs/r/storage_default_object_acl.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: "google"
+page_title: "Google: google_storage_default_object_acl"
+sidebar_current: "docs-google-storage-default_object-acl"
+description: |-
+  Creates a new default object ACL in Google Cloud Storage.
+---
+
+# google\_storage\_object\_acl
```

Comment:
This should be `google_storage_default_object_acl`.

Reply:
Fixed